src/Business/Detection/Fetching/Models/PDetection.cs

```diff
@@ -7,7 +7,7 @@ public class PDetection
     {
         public int? Id { get; set; }
 
-        public float? Score { get; set; }
+        public MScoreFilter? ScoreFilter { get; set; } = new MScoreFilter();
```

Comment:
Would the name `Score` be more relevant here?

Reply:
done